docs/source/markdown/podman-image-diff.1.md

```diff
@@ -1,30 +1,45 @@
 % podman-image-diff(1)
 
 ## NAME
+
 podman-image-diff - Inspect changes on an image's filesystem
 
 ## SYNOPSIS
+
```

Comment:
I think our validate script does not allow an extra newline after the heading.

Reply:
It does work, but for the uniformity of all manpages I'll specify in the MANPAGE_SYNTAX, that there shouldn't be a new line.

Reply:
@Procyhon I am using markdownlint, which is a vscode extension and they suggest using a new line https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022
I think it just makes the markdown source code more readable

Reply:
But let me know what is the final verdict on this